spec/models/badge_spec.rb

```diff
@@ -80,4 +80,21 @@
       it { is_expected.to be true }
     end
   end
+
+  describe '.i18n_name' do
+    it 'transforms to lower case letters, and replaces spaces with underscores' do
+      expect(Badge.i18n_name('Basic User')).to eq('basic_user')
+    end
+  end
+
+  describe '.display_name' do
+    it 'fetches from translations when i18n_name key exists' do
+      expect(Badge.display_name('basic_user')).to eq('Basic')
+      expect(Badge.display_name('Basic User')).to eq('Basic')
```

Comment:
I usually prefer to avoid repetition in the code base by looping instead. 

```
["basic_user", "Basic User"].each { |name| expect(Badge.display_name(name)).to eq('Basic')  }
```

Reply:
👍 on looping, but I think it may overcomplicate the test case here since there are only two. I don't feel strongly about it tho.

Reply:
Yeah I am not a fan of a loop of 2, 20 yeah, 2 is overkill imo, this reads fine to me